src/app/components/apparatus-entry/apparatus-entry-detail/apparatus-entry-detail.component.html

```diff
@@ -0,0 +1,104 @@
+<div class="card app-detail-container">
```

Comment:
This component is very large. Is it possible to modularize it a bit?

Reply:
I would manage modularization in a new distinct feature.